frb_codegen/src/others.rs

```diff
@@ -35,6 +35,8 @@ lazy_static! {
         vec!["store_dart_post_cobject".to_string()];
 }
 
+pub const CARGO_PKG_VERSION: &str = env!("CARGO_PKG_VERSION");
+
 pub const CODE_HEADER: &str = "// AUTO GENERATED FILE, DO NOT EDIT.
```

Comment:
e.g. 

```
pub const CODE_HEADER: &str = format!("// AUTO GENERATED FILE, DO NOT EDIT.
// Generated by `flutter_rust_bridge` @ {}.", env!("CARGO_PKG_VERSION"));

Reply:
Thing is `format!` is not a `const` function so it raises a compile-time error. I'm gonna search for a way that doesn't require to add a dependency like e.g. [const_format](https://docs.rs/const_format/latest/const_format/).

Reply:
Then what about

```
pub fn code_header() -> String { ... }

Reply:
done in 9da58a7be169b7d669486b892d4a4d472324264a

Reply:
I switched CODE_HEADER visibility to `private` so that people are more inclined to use `fn code_header()` instead.